utils/telemetry/tracker.go

```diff
@@ -0,0 +1,57 @@
+package telemetry
+
+import (
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+var (
+	segmentWriteKey string //value will be injected at build-time
+)
+
+// REALMC-7243 same as the new cli Segment tracking
```

Comment:
can get rid of these comments (but also as a sidenote: i'm noticing a lot of missing docstrings on otherwise exported package contents).  can we look into why `lint` isn't failing?  i see a "Command failed: no files found to be parsed" in the evg output, not sure if that just always shows up or not, but let's investigate (and add docstrings)